frontend/src/pages/mainPage.js

```diff
@@ -184,15 +187,15 @@ function MainPage() {
         const selectedPage = event.selected;
         setCurrentPage(selectedPage);
         getPaginatedNews(selectedPage);
+        scrollToTarget();
     };
 
     const [selectedSport, setSelectedSport] = useState("all");
-    const element_height = 100
-    const element_width = 350
     return (
         <>
             <div className="showcase">
-                <section>
+                {!noLatestNews ? (
```

Comment:
See? You basically get: "no no latest news" here

While not a mistake - not goodlooking either